### PR TITLE
Update average.js

### PR DESCRIPTION
**Pull request by selensuda**
Add function definition summary.

Addresses Issue: #theRelatedIssueNumber

- [] Running `yarn lint` does not trigger any linter errors
- [] The test of the method you have fixed is passing
- [] You have written a new skeleton method for someone else to work on!
- [] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/average.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
-// Write a function that returns the average of the numbers in an array
-
+/**
+ * Taes an array and returns the average of the numbers in it
+ * that are in one array but not in the other one.
```
Comment (ktilcu) [inline lib/average.js]: This line seems out of place.

File: lib/average.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
-// Write a function that returns the average of the numbers in an array
-
+/**
+ * Taes an array and returns the average of the numbers in it
```
Comment (ktilcu) [inline lib/average.js]: `* Takes` - 😛

**Comment by selensuda** [comment]
@ktilcu Fixed the line and grammar :) Sorry about that!

**Comment by ktilcu** [comment]
@selensuda no Prob. Why don't you pull in those other doc changes to this PR too.

**Comment by BlakeGuilloud** [comment]
I'm going to go ahead a merge this. @selensuda if you wish to bring in more documentation, feel free to open up a new pull request, but try to keep all of your changes in a single branch! Thanks!!

**Pull request merged**